lib/rollbax/logger.ex

```diff
@@ -89,12 +89,14 @@ defmodule Rollbax.Logger do
     Logger.compare_levels(lvl, min) != :lt
   end
 
-  defp post_event(level, {Logger, message, event_time, meta}, keys) do
+  defp post_event_unless_whitelisted(level, {Logger, message, event_time, meta}, keys, whitelist) do
     event_unix_time = event_time_to_unix(event_time)
     message = message |> prune_chardata() |> IO.chardata_to_string()
-    meta = Keyword.take(meta, keys) |> Enum.into(%{})
-    body = Rollbax.Item.message_to_body(message, meta)
-    Rollbax.Client.emit(level, event_unix_time, body, %{}, %{})
+    unless Enum.any?(whitelist, fn(r) -> message =~ r end) do
```

Comment:
Instead of using a literal `fn` with a cryptic one-letter argument like `r`, we could use `&(message =~ &1)`. Wdyt?